apps/server/src/modules/learnroom/learnroom.module.ts

```diff
@@ -48,6 +61,13 @@ import { CommonCartridgeFileValidatorPipe } from './utils';
 		LoggerModule,
 		TaskModule,
 		CqrsModule,
+		SystemModule,
```

Comment:
I don't think you use most of these modules. I did not see any import of any module